### PR TITLE
Set up a "Hello Microverse" project

### PR DESCRIPTION
**Pull request by Abdelaziz-Mahdi**
<h2>Files created inside this branch are: </h2>
1- .gitignore file.

2- linter file.
3- Descriptive README file.
4- html file that includes a header with the text "Hello Microverse!".
5- .css file that includes one class with styles for the header.
6- test.md file with the text "This file should be ignored by git".

### CODE REVIEW

File: test.md [hunk 1]
```diff
@@ -0,0 +1 @@
+"This file should be ignored by git"
```
Comment (anagudelogu) [inline test.md]: - [ ] You did a great job following the requirements! 💯  However, please make sure that this file is ignored by git. Consider fixing this one! 🤓 🛠️

**Review by anagudelogu — CHANGES_REQUESTED** [review]
# STATUS:  CHANGES REQUIRED  ♻️ 🚧 

Hi @Abdelaziz-Ali 
I'm Andres.


While you have done a great job working on this milestone 💻 , there are some issues that you still need to work on to go to the next project. ♻️ 

You are almost there, keep up the good work! 🚀 



## Highlights
✅ Great use of GitHub flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Descriptive PR title, summary and commit messages. 💬 

## Required Changes ♻️

- [ ] I noticed that you didn't install linters locally in your project, please consider following this [entire step-by-step tutorial](https://github.com/microverseinc/linters-config/tree/master/html-css) on how to add linters to your HTML and CSS projects (You're missing `.hintrc` and `.stylelintrc.json` files). 🤓 🛠️ 

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by emmiiorji — CHANGES_REQUESTED** [review]
Hi @Abdelaziz-Ali, 

# Status: Changes Required ♻️

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

A little bit more

 ## Good Highlights✔️

- Linter checks are passing
- Good commit messages
- Good use of GitHub flow


 ## Required Changes ♻️

- You have done great by adding the `test.md` file to `.gitignore`. However, the file is still committed. This is because it requires a commit where the file gets deleted in order to be actually deleted on GitHub. You can get around this by:
    - deleting the file locally and removing it from `.gitignore`
    - committing and pushing the changes to github
    - creating the file again and adding it to `.gitignore`
    - committing and pushing your changes again

_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by frank1738 — APPROVED** [review]
Hi @Abdelaziz-Ali , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

✅ Correct use of Github workflow
✅ No linter error
✅ Good PR Description 👍 

## Optional suggestions

N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**